docs/source/development/index.rst

```diff
@@ -14,30 +12,180 @@ The key components of AiiDAlab are:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+First, replace the system home app with a local copy based on your fork as described in the instructions for development of the home app.
+The clone your fork of the aiidalab package into a location within user space of your choice (e.g. ``$HOME/aiidalab``) and create a symbolic link within the home app to that location:
+
+.. code-block:: console
+
+    ~$ git clone git@github.com:<USERNAME>/aiidalab.git
+    ~$ cd apps/home/
+    ~/apps/home$ ln -s $HOME/aiidalab/aiidalab
+
+In this way, you can test the function of the home app directly against
+your development version of aiidalab.
+
+Once satisfied with the change set, push it to your fork and create a
+(draft) pull request.
+This will ensure that the unit and integration tests are executed.
+
+How to make a new release (aiidalab and aiidalab-home)
+------------------------------------------------------
+
+#. Switch to ``develop`` branch.
+#. Make a new release branch e.g. ``release/1.0.0a2``.
+
+.. code-block:: console
+
+       $ git checkout -b release/1.0.0a2
+
+#. Increment ``__version__`` in ``__init__.py`` AND ``setup.json``
+   (setup.py).
+#. Commit the changes with the commit message: "Prepare release
+   1.0.0a2".
+#. Push the changes to GitHub.
+#. Create a PR from ``release/1.0.0a2`` branch to the ``master`` branch.
+#. Once the tests are passed, merge the PR via merge commit (do not use
+   rebase, it will recreate all the commits' sha!).
+#. Go to the command line and pull the ``master`` branch.
+#. Delete ``release/1.0.0a2`` branch locally.
+
+.. code-block:: console
+
+        $ git branch -d release/1.0.0a2
+
+#. Make a new annotated tag. Commit message: "Release 1.0.0a2"
+
+.. code-block:: console
+
+        $ git tag -a 1.0.0a2
+
+#. Push the new tag to GitHub: 
+
+.. code-block:: console
+
+       $ git push --tags
+
+#. List all new changes (with links to PRs) while making a release on
+   GitHub.
+#. Remove the old dist folder.
+
+.. code-block:: console
+
+       $ rm -r dist
+
+#. Create a new ``dist`` folder.
+
+.. code-block:: console
+
+        $ python setup.py sdist
+
+#. Upload things on test.pypi.org 
+
+.. code-block:: console
+
+       $ twine upload --repository-url https://test.pypi.org/legacy/ dist/*
+
+#. Check on test.pypi.org that everything is alright.
+#. Upload things on pypi.org:
+
+.. code-block:: console
+
+        $ twine upload dist/*
+
+#. Merge ``master`` back to ``develop`` and push the ``develop`` branch
+   to GitHub.
+
+.. code-block:: console
+
+        $ git checkout develop && git merge master && git push
+
+-  Get latest version of dependencies:
+    ``pip install pur    pur``
+    Make sure to review changes!
+-  Upload package
+
+   ``pip install --upgrade twine wheel    rm -rf build/ dist/    python setup.py sdist bdist_wheel    # for testing use    # twine upload --repository-url https://test.pypi.org/legacy/ dist/*    twine upload dist/*``
```

Comment:
I am not sure why this is needed, but the command to upload things on twine is already shown above.

Reply:
I copied this from the wiki and I think the original description came from you. It looks like some of the formatting got messed up. Since you create the releases for the aiidalab package, is this the correct approach? Otherwise I will revise that section completely.

Reply:
The text that I originally provided ends at the line:
```
        $ git checkout develop && git merge master && git push
```
See [here](https://github.com/aiidalab/aiidalab/wiki/Home/_compare/e209dd13a227c49370f5c45f6af1fbeea829fa14...7857906579662f16d7656a426673110b94e255b0).

Those instructions are still correct and I still use them every time I make a new release.